tdvt/tdvt/logicaltests/expected/connection_test/expected.setup.staples.connection.test.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<results>
+  <test name='/Users/lpetschauer/PycharmProjects/connector-plugin-sdk/tdvt/tdvt/logicaltests/setup/connection_test/setup.staples.connection.test.simple.xml'>
+    <sql>
+      SELECT &quot;Staples&quot;.&quot;Ship Priority&quot; AS &quot;Ship Priority&quot;&#10;FROM &quot;Staples&quot;&#10;GROUP BY 1&#10;    </sql>
+    <query-time>
+      122.000&#10;    </query-time>
+    <table>
+      <schema>
+        <column>[Staples.postgres].[Ship Priority]</column>
+      </schema>
+      <tuple>
+        <value>&quot;0&quot;</value>

```

Comment:
The switch from `Discount` to `Ship Priority` doesn't seem like it'll do a very good job of capturing if the data is loaded correctly in the Staples table or not since the only value is `0`.  That was my impression of the intent around these tests?

Reply:
You're right, I wasn't thinking. Fixed in the commit below.